oracle/oracle.ipynb

```diff
@@ -0,0 +1,571 @@
+    "# If running in google colab, this is needed.\n",
+    "\n",
+    "!pip install cadcad matplotlib pandas numpy"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "6cd1b04c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import dependancies\n",
+    "\n",
+    "# Data processing and plotting libraries\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "from random import normalvariate\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "# cadCAD specific libraries\n",
+    "from cadCAD.configuration.utils import config_sim\n",
+    "from cadCAD.configuration import Experiment\n",
+    "from cadCAD.engine import ExecutionContext, Executor"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "54a15650",
+   "metadata": {},
+   "source": [
+    "# Historical modeling"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "9d86f390",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import historical_model as hm\n",
+    "MONTE_CARLO_RUNS = 1\n",
+    "SIMULATION_TIMESTEPS = len(hm.btc_diff_data)\n",
+    "\n",
+    "sim_config = config_sim(\n",
+    "    {\n",
+    "        'N': MONTE_CARLO_RUNS,\n",
+    "        'T': range(SIMULATION_TIMESTEPS)\n",
+    "    }\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5d06cb70",
+   "metadata": {},
+   "source": [
+    "Running just the normal configuration (no volatility, historical data since 2016)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "15bf36da",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "                  ___________    ____\n",
+      "  ________ __ ___/ / ____/   |  / __ \\\n",
+      " / ___/ __` / __  / /   / /| | / / / /\n",
+      "/ /__/ /_/ / /_/ / /___/ ___ |/ /_/ /\n",
+      "\\___/\\__,_/\\__,_/\\____/_/  |_/_____/\n",
+      "by cadCAD\n",
+      "\n",
+      "cadCAD Version: 0.4.28\n",
+      "Execution Mode: local_proc\n",
+      "Simulation Dimensions:\n",
+      "Entire Simulation: (Models, Unique Timesteps, Params, Total Runs, Sub-States) = (1, 65240, 0, 1, 7)\n",
+      "     Simulation 0: (Timesteps, Params, Runs, Sub-States) = (65240, 0, 1, 7)\n",
+      "Execution Method: local_simulations\n",
+      "Execution Mode: single_threaded\n",
+      "Total execution time: 4.84s\n"
+     ]
+    }
+   ],
+   "source": [
+    "from cadCAD import configs\n",
+    "del configs[:]\n",
+    "\n",
+    "experiment = Experiment()\n",
+    "\n",
+    "experiment.append_configs(\n",
+    "    sim_configs=sim_config,\n",
+    "    initial_state=hm.initial_state,\n",
+    "    partial_state_update_blocks=hm.partial_state_update_blocks,\n",
+    ")\n",
+    "\n",
+    "exec_context = ExecutionContext()\n",
+    "run = Executor(exec_context=exec_context, configs=experiment.configs)\n",
```

Comment:
I'm getting this error when I try to run it :/ I checked that my `cadcad` package version is the latest. Did you run into anything similar?
<img width="469" alt="image" src="https://user-images.githubusercontent.com/11261382/235279151-1165c38f-ba91-4a82-805e-958eacae992c.png">